Ports/kakoune/patches/0002-Set-Kakoune-s-version.patch

```diff
@@ -0,0 +1,26 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: demostanis <demostanis@protonmail.com>
+Date: Sat, 10 Sep 2022 23:25:05 +0200
+Subject: [PATCH] Set Kakoune's version
+
+This looks better than 'unknown'
+---
+ src/Makefile | 2 +-
+ 1 file changed, 1 insertion(+), 1 deletion(-)
+
+diff --git a/src/Makefile b/src/Makefile
+index 801a2e2..582a32d 100644
+--- a/src/Makefile
++++ b/src/Makefile
+@@ -38,7 +38,7 @@ ifneq (,$(sanitize_suffix))
+     suffix := $(suffix).san_$(sanitize_suffix)
+ endif
+ 
+-version ?= $(shell if [ -f .version ]; then cat .version; elif  [ -d ../.git ]; then git describe --tags HEAD; else echo "unknown"; fi)
++version ?= on SerenityOS
```

Comment:
I'd argue that we shouldn't touch the version at all, this is really just unnecessary patch noise. `unknown` is fine for now since we aren't actually building any specific release, and after we change to a tagged release there is no patch to change either.